app/views/account_sign_up/new.html.erb

```diff
@@ -15,13 +15,6 @@
             </ul>
           </div>
         <% end %>
-        <div class="form-group">
-          <%= f.label :title, 'Title', class: 'col-md-2 control-label required' %>
-          <div class="col-md-10">
-            <%= f.text_field :title, class: 'form-control' %>
-            <span class="help-block">A human-friendly title for the repository. This can be changed later.</span>
```

Comment:
Yeah, the problem is that whatever **intent** never got **implemented**.  Once submitted, `@account.title` is not used.